.github/workflows/periodic.yaml

```diff
@@ -15,8 +15,15 @@
 name: Periodic
 
 on:
-  schedule:
-  - cron:  '0 */12 * * *'
+  pull_request:
+    branches:
+      - development
+      - "[0-9]+.[0-9]+.x"
+
+  # Run CI also on push to master
+  push:
+    branches:
+      - master
```

Comment:
That makes it non-periodic 🤔

Reply:
@TomerShor yes, it is just for testing purposes

Reply:
reverted :)